zebra-network/src/constants.rs

```diff
@@ -57,7 +57,7 @@ pub const SYNC_FANOUT: usize = 4;
 ///
 /// We want enough buffer to support concurrent fanouts, a sync download,
 /// an inbound download, and some extra slots to avoid contention.
-pub const PEERSET_BUFFER_SIZE: usize = MAX_GET_ADDR_FANOUT + SYNC_FANOUT + 20;
+pub const PEERSET_BUFFER_SIZE: usize = MAX_GET_ADDR_FANOUT + SYNC_FANOUT + 50;
```

Comment:
Why is Zebra so sensitive to buffer sizes?
It seems like it's just CI, it's much harder to reproduce locally.